python/ql/lib/semmle/python/frameworks/data/internal/ApiGraphModels.qll

```diff
@@ -70,8 +70,8 @@ private module API = Specific::API;
 
 private module DataFlow = Specific::DataFlow;
 
-private import Specific::AccessPathSyntax
 private import ApiGraphModelsExtensions as Extensions
+import codeql.dataflow.internal.AccessPathSyntax
```

Comment:
Can this import be made private?

```suggestion
private import codeql.dataflow.internal.AccessPathSyntax
```

Reply:
Yes, but it requires an extra import in `ApiGraphModelsSpecific.qll`, but that's ok.